lib/src/animation_id.dart

```diff
@@ -39,17 +43,27 @@ class AnimationId {
     try {
       final animationId = AnimationId.parse(id);
       return animationId;
-    } catch (e) {
+    } catch (e, s) {
+      debugPrintStack(label: e.toString(), stackTrace: s);
```

Comment:
I think this print should be behind a `verbose` flag 🤔

Reply:
Nice catch! Actually I forgot to remove this, I used it when I was debugging. I have removed it.